File: test1/scala/workflows/abhi_test_wf1/code/src/main/scala/graph/Target_1.scala
package graph

import io.prophecy.libs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import config.ConfigStore._

object Target_1 {

  def apply(spark: SparkSession, in: DataFrame): Unit = {
    Config.fabricName match {
      case "AbhishekSharedFabric" =>
        in.write
          .format("csv")
          .option("header", true)
          .option("sep",    ",")
          .mode("overwrite")
          .save("dbfs:/tmp/e2e/dataset_out_14814")
      case _ =>
        throw new Exception("No valid dataset present to read fabric")
    }
  }

}

File: uitesting/e2etestsmain2/pipelines/Workflow-Automated-GITHUB-1654086151641/code/job/graph/Target_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Target_1(spark: SparkSession, in0: DataFrame):
    if Config.fabricName == "dev":
        in0.write\
            .option("header", True)\
            .option("sep", ",")\
            .mode("overwrite")\
            .option("separator", ",")\
            .option("header", True)\
            .csv("dbfs:/tmp/e2e/dataset_out_99755")
    else:
        raise Exception("No valid dataset present to read fabric")
